cloud/linode/mock_client_test.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/linode/linode-cloud-controller-manager/cloud/linode (interfaces: Client)
+// Source: github.com/linode/linode-cloud-controller-manager/cloud/linod (interfaces: Client)
 
-// Package linode is a generated GoMock package.
+// Package client is a generated GoMock package.
```

Comment:
Why is this renamed? I know it's just a comment, but it's weird - did you change the mockgen calls or something? If you revert this, will it regenerate it again?

Reply:
Finally resolved this problem. There was a PR #186 which introduced the double mock_client_test.go. I have checked this with @AshleyDumaine and the result is that I have removed both these files in favor of mocks/mock_client.go which is now the only needed and it is generated.